common/ostream.h

```diff
@@ -78,7 +78,7 @@ namespace llvm {
 // this, and so we want to prioritize accordingly.
 //
 // It would be slightly cleaner for LLVM itself to provide this overload in
-// `raw_os_ostream.h` so that we wouldn't need to inject into its namespace, but
+// `raw_os_ostream.h` so that we wouldn't need to inject it into its namespace, but
 // supporting `std::ostream` isn't a priority for LLVM so we handle it locally
```

Comment:
I think the original is closer but still not quite right... Maybe something like this would be overall more clear?

```suggestion
// `raw_os_ostream.h` so that we wouldn't need to inject into LLVM's namespace, but
// supporting `std::ostream` isn't a priority for LLVM so we handle it locally
```

Reply:
It seems that there is a pronoun problem here even if we replace "its" with "LLVM's".

Reply:
I agree with Chandler's suggestion here. What we would like to avoid is injecting into LLVM's namespace.